src/testsuite/vec_perf_f32.c

```diff
@@ -75,12 +76,14 @@ int timed_is_f32 (void)
 
 int timed_fpclassify_f32 (void)
 {
+#ifndef __clang__
```

Comment:
why? add a comment.

Reply:
// Clang does not define vec_test_data_class and friends.